src/briefcase/bootstraps/pursuedpybear.py

```diff
@@ -0,0 +1,150 @@
+from briefcase.bootstraps.base import BaseGuiBootstrap
+
+
+class PursuedPyBearGuiBootstrap(BaseGuiBootstrap):
+    def app_source(self):
+        return """\
+import importlib.metadata
+import os
+import sys
+
+import ppb
+
+
+class {{ cookiecutter.class_name }}(ppb.Scene):
+    def __init__(self, **props):
+        super().__init__(**props)
+
+        self.add(
+            ppb.Sprite(
+                image=ppb.Image("{{ cookiecutter.module_name }}/resources/{{ cookiecutter.app_name }}.png"),
+            )
+        )
+
+
+def main():
+    # Linux desktop environments use app's .desktop file to integrate the app
+    # to their application menus. The .desktop file of this app will include
+    # StartupWMClass key, set to app's formal name, which helps associate
+    # app's windows to its menu item.
+    #
+    # For association to work any windows of the app must have WMCLASS
+    # property set to match the value set in app's desktop file. For PPB this
+    # is set using environment variable.
+
+    # Find the name of the module that was used to start the app
+    app_module = sys.modules["__main__"].__package__
+    # Retrieve the app's metadata
+    metadata = importlib.metadata.metadata(app_module)
+
+    os.environ["SDL_VIDEO_X11_WMCLASS"] = metadata["Formal-Name"]
+
+    ppb.run(
+        starting_scene={{ cookiecutter.class_name }},
+        title=metadata["Formal-Name"],
+    )
+"""
+
+    def pyproject_requires(self):
+        return """
+    "ppb~=1.1",
+"""
+
+    def pyproject_test_requires(self):
+        return """
+{% if cookiecutter.test_framework == "pytest" %}
+    "pytest",
+{% endif %}
+"""
+
+    def pyproject_table_macOS(self):
+        return """
+universal_build = true
+requires = [
+    "std-nslog~=1.0.0",
+]
+"""
+
+    def pyproject_table_linux(self):
+        return """
+requires = [
+]
+"""
+
+    def pyproject_table_linux_system_debian(self):
+        return """
+system_requires = [
+# ?? FIXME
```

Comment:
As much as possible, I tried to re-create the existing behavior (such as including these `FIXME` comments)....so, if we want to make more targeted changes to the templates based on the GUI, now is definitely a good time to recommend those.